assets/prototype/domain/shared/data/queries/currentUser/useFetchCurrentUser.ts

```diff
@@ -0,0 +1,16 @@
+import { useQuery } from '@apollo/react-hooks';
+import { GET_CURRENT_USER } from './';
+import { FetchEntitiesResult } from '../../../../eventEditor/data/queries/types';
+import { Viewer } from '../../../../../application/valueObjects/config/types';
+
+const useFetchCurrentUser = (): FetchEntitiesResult<Viewer> => {
+	const { data, error, loading } = useQuery<Viewer>(GET_CURRENT_USER);
+
+	return {
+		data,
+		error,
+		loading,
```

Comment:
Should these handled by the toaster, or that would happen on a different step?

Reply:
Any consumer can make use of that but we haven't added that to toaster yet.

Reply:
Any consumer can make use of that but we haven't added that to toaster yet.